test/jni/Android.mk

```diff
@@ -458,7 +458,6 @@ LOCAL_SRC_FILES := \
 	 ../unittest/layers/unittest_layers_flatten.cpp \
 	 ../unittest/layers/unittest_layers_activation.cpp \
 	 ../unittest/layers/unittest_layers_addition.cpp \
-	 ../unittest/layers/unittest_layers_addition_cl.cpp \
```

Comment:
Question) is there any reason to remove this from unittest_layers build ?

Reply:
This `../unittest/layers/unittest_layers_addition_cl.cpp` should be added after #2752.
If not, NDK build returns error, for addition layer for cl kernel is not implemented in the current upstream. I wanted to make upstream run as it is.
By the way... it would be better to change this PR [Wait for #2752] and remove this change.

Reply:
i see, thank you for detail explain 

```By the way... it would be better to change this PR [Wait for https://github.com/nnstreamer/nntrainer/pull/2752], while removing this change.```  
Yes i would be good!! or i think you can simply rebase #2752 or cherry pick some commit's ( if there are some dependency)

rebase sample command
``` git pull --rebase upstream refs/pull/2752/head: ```

Reply:
Sounds bood! I will rebase #2752. Thank you ☺

Reply:
@DonghakPark 
I tested rebasing on #2752, but the PR itself is not well rebased on the main; It makes this PR jammed.
Instead of rebasing on it, I updated the `Android.mk` to leave line you mentioned as commented out. Could you please check the update?
If #2752 is merged or it resolves the rebasing issue, I will update this PR as well.

Reply:
If so, it seems to be good. It may reduce inconvenience